code/game/objects/items/devices/multitool.dm

```diff
@@ -179,3 +179,22 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
+
+/obj/item/multitool/makeshift
+	name = "makeshift multitool"
+	desc = "As crappy as it is, its still mostly the same as a standard issue Nanotrasen one."
+	icon = 'icons/obj/improvised.dmi'
+	icon_state = "multitool_makeshift"
+	toolspeed = 2
+
+/obj/item/multitool/makeshift/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(prob(5))
+		to_chat(user, span_danger("[src] crumbles apart in your hands!"))
+		qdel(src)
+		return
+	else if(prob(5))
+		user.rad_act(20)
+		to_chat(user, span_userdanger("[src] breaks down and emits dangerous rays!"))
+		src.tool_behaviour = 0
+		return
```

Comment:
Update the description here to say that it's busted, also have there be a check if it's functional or not before returning parent